ui/src/pages/backoffice/Series/SeriesSearch/SeriesListEntry.js

```diff
@@ -33,6 +33,17 @@ export class SeriesListEntry extends Component {
 
   renderRelations = () => {
```

Comment:
@kprzerwa 
I think we should remove all links to search query here. For example, I went to one that had 5 editions and the link is: `.../documents?q=pid: 6vptg-ty662 OR pid: 9msba-sxt46 OR pid: r43kz-hdv89 OR pid: t24n3-ht166 OR pid: hnsq8-krm96`.
The list of `OR` can be potentially massive.

IMHO, it is important to show the text, e.g `5 editions`, then when you go inside the details page, you can see the full list in the relations boxes...

So in this case, I would just count as suggested just above and show `10 literatures` text and remove the link.

WDYT?